src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
I've I think resolved this in the RC: https://github.com/Pablissimo/SonarTsPlugin/commit/25a8523cae70a0626d0a7eb2ab2d9c0fbab42a56

Appears to fix my issue of a local tslint run's output being presented to the plugin directly via the new ```sonar.ts.tslint.outputPath``` parameter, would be curious to see if it resolves it in all situations.